src/features/locators/Locator.tsx

```diff
@@ -131,8 +137,11 @@ export const Locator: React.FC<Props> = ({ element, currentPage, searchState, de
 
     return (
       <span onClick={handleClick}>
-        @UI(
-        <span className="jdn__xpath_item-locator">&quot;{locator.output}&quot;</span>)
+        {annotationType}(
+        <span className="jdn__xpath_item-locator">
+          {isUiAnnotationType() ? "" : isXpathLocatorType() ? "xpath = " : "css = "}&quot;{locator.output}&quot;
```

Comment:
Maybe while you update this code, move texts to constants?

Reply:
Moved it to a separate helper function, not sure if it’s really needed in this case 🙂